package erratum

const testVersion = 2

func Use(MyFunc func(Resource, error) string, message string) string {
	return ""
}
